lib/roo/excelx/cell/number.rb

```diff
@@ -66,6 +66,16 @@ def generate_formatter(format)
           when '##0.0E+0' then '%.1E'
           when "_-* #,##0.00\\ _€_-;\\-* #,##0.00\\ _€_-;_-* \"-\"??\\ _€_-;_-@_-" then number_format('%.2f', '-%.2f')
```

Comment:
This expression is similar but it isn't defended with a spec so I can't tell if I could handle it too. It also appears to discard the € which I in our case we want in the formatted value.